README.md

```diff
@@ -20,3 +20,4 @@ A curated list of Madagascar related projects.
 
 ## Fun
 - [rasolofonirina/quotes-gasy](https://github.com/rasolofonirina/quotes-gasy) - A list of famous (or not) quotes from various authors.
+- [tokyRT/Gasy-quotes-explorer](https://github.com/tokyRT/Gasy-quotes-explorer) - A web UI for [rasolofonirina/quotes-gasy's repository](https://github.com/rasolofonirina/quotes-gasy).
```

Comment:
```suggestion
<!--lint ignore double-link-->
- [tokyRT/Gasy-quotes-explorer](https://github.com/tokyRT/Gasy-quotes-explorer) - A web UI for [rasolofonirina/quotes-gasy's repository](https://github.com/rasolofonirina/quotes-gasy).
```

Reply:
This is a mainstream bug so we have to use this workaround

Reply:
I think the description should not be too restricted